Drop-In/build.gradle

```diff
@@ -47,7 +47,9 @@ android {
         warning 'GradleCompatible'
         textReport true
         textOutput 'stdout'
-        disable 'InvalidPackage' // The Cardinal SDK is referencing the `javax.naming` package but it is unavailable
+        // RestrictedApi - Accessing Braintree Core SDK non-public methods
+        // InvalidPackage - The Cardinal SDK is referencing the `javax.naming` package but it is unavailable
+        disable 'RestrictedApi', 'InvalidPackage'
```

Comment:
random: I wonder if Cardinal actually needs `javax.naming` to work

Reply:
Actually looks like they don't use it anymore or it's not causing a lint error - removed that option 👍